recursion/gnark-ffi/src/groth16.rs

```diff
@@ -26,9 +33,46 @@ pub struct Groth16Proof {
 }
 
 impl Groth16Prover {
-    /// Creates a nejw verifier.
+    /// Starts up the Gnark server using Groth16 on the given port.
     pub fn new() -> Self {
-        Groth16Prover
+        let manifest_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        let gnark_dir = manifest_dir.join("../gnark");
+        let port = env::var("HOST_PORT").unwrap_or_else(|_| "8080".to_string());
+        let port_clone = port.clone();
+
+        // Start the Gnark server in a separate thread.
+        let rt = Runtime::new().unwrap();
```

Comment:
Also, instead of tokio/rt, can we just use a simple process/thread? Async should be unecessary.

Reply:
updated